src/sentry/web/frontend/debug/mail.py

```diff
@@ -430,3 +431,39 @@ def access_approved(request):
             'team': team,
         },
     ).render()
+
+
+@login_required
+def confirm_email(request):
+    email = request.user.emails.first()
+    email.set_hash()
+    email.save()
+    return MailPreview(
+        html_template='sentry/emails/confirm_email.html',
+        text_template='sentry/emails/confirm_email.txt',
+        context={
+            'confirm_email': 'foo@example.com',
+            'user': request.user,
+            'url': absolute_uri(reverse(
+                'sentry-account-confirm-email',
+                args=[request.user.id, email.validation_hash]
+            )),
+            'is_new_user': True,
+        },
+    ).render()
+
+
+@login_required
+def recover_account(request):
+    return MailPreview(
+        html_template='sentry/emails/recover_account.html',
+        text_template='sentry/emails/recover_account.txt',
+        context={
+            'user': request.user,
+            'url': absolute_uri(reverse(
+                'sentry-account-confirm-email',
+                args=[request.user.id, 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX']
```

Comment:
yeah didn't feel like it was necessary to generate a hash just for the debug view since i was just making sure the layout looked ok